.gitignore

```diff
@@ -18,4 +20,8 @@ Cargo.lock
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-#.idea/
\ No newline at end of file
+#.idea/
+
+# Added by cargo
+
+/target
```

Comment:
What's the difference between `/target` here and `target/` at L6?

Reply:
This was auto-added by rust-analyzer I think